controllers/admin/AdminImportController.php

```diff
@@ -1428,32 +1992,34 @@ protected function categoryImportOne($info, $default_language_id, $id_lang, $for
         //copying images of categories
         if (isset($category->image) && !empty($category->image)) {
             if (!(AdminImportController::copyImg($category->id, null, $category->image, 'categories', !$regenerate))) {
-                $this->warnings[] = $category->image.' '.$this->trans('cannot be copied.', array(), 'Admin.Advparameters.Notification');
+                $this->warnings[] = $category->image.' '.$this->trans('cannot be copied.',
```

Comment:
In other languages, the image name may go at a different place. We should add it as a parameter.

Reply:
+1, thanks @Quetzacoalt91 for spotting this!

Reply:
Maybe this is outside the scope of this refacto ?